exercises/all-your-base/test/test_all_your_base.c

```diff
@@ -0,0 +1,200 @@
+#include "vendor/unity.h"
+#include <stddef.h>
+#include <stdint.h>
+#include "../src/all_your_base.h"
+
+#include <stdio.h>
+
+void fill_array(int16_t data[], size_t data_length, int16_t array[],
+                size_t array_length)
```

Comment:
This is basically right justifying the array, right? That wasn't clear to me from the name alone. So, we could rename this, but I guess I'm left wondering why we need to right justify it in the first place. Can't we just let the data be left justified? I think that would take some of the mystery out of the test setup. As it stands, I think people will have some trouble figuring out exactly what format the input and output will be based without digging into the details of these helpers. I think we can remove some of the complexity from the helpers and make it more "what you see is what you get".